extensions/Worker.Extensions.Storage.Blobs/src/Constants.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    internal static class Constants
+    {
+        internal const string BlobExtensionName = "AzureStorageBlobs";
+        internal const string Connection = "Connection";
+        internal const string ContainerName = "ContainerName";
+        internal const string BlobName = "BlobName";
+
+        // Media content types
+        internal const string JsonContentType = "application/json";
```

Comment:
I think this constant is available: https://learn.microsoft.com/en-us/dotnet/api/system.net.mime.mediatypenames.application.json?view=net-7.0

Reply:
Yeah I tried using that before but we're on .net standard 2.0 and `Json` was added in 2.1

When I use `MediaTypeNames.Application.Json` I get this error: `MediaTypeNames.Application' does not contain a definition for 'Json' `

We can maybe change the target to <TargetFramework>netstandard2.1</TargetFramework> but I'm not sure of the impact of that

Reply:
We can't move to `netstandard2.1` here as it would impact our .NET Framework support. For this simple change/API usage, I would also not consider any of the other options, as the benefit here wouldn't justify the added complexity.